src/SecretKey.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PharIo\GnuPG;
+
+/**
+ * @todo define what is needed here.
+ */
+class SecretKey
```

Comment:
This should probably be renamed to `SecretKeyData`. And a new `SecretKey` could be created that contains this class as one property.